lumen/target.py

```diff
@@ -344,6 +340,8 @@ def _construct_cards(self):
         controls, all_transforms = [], []
         linked_views = None
         for facet_filters in self.facet.filters:
+            if self.views is None:
+                raise ValueError(f"Ensure that the target '{self.title}' declares a 'views' field.")
```

Comment:
Let's just validate that a target has to declare views in `Target.from_spec` and simple skip `_construct_cards` entirely if no views are declared.

Reply:
I think [a9106fb](https://github.com/holoviz/lumen/pull/300/commits/a9106fb2227e232772c0864e0ff23e506ebf75d7) does what you are asking for.